src/test/steps/IdamUserManager.ts

```diff
@@ -50,11 +50,15 @@ export class IdamUserManager {
     }
   }
 
+  async clearAndKeepOnlyOriginalUser(): Promise<void> {
+    this.users = new Set([...this.users][0]);
```

Comment:
new Set(this.users[0])?

Reply:
creates a new set with the first element from the previous users.

Reply:
i guess i can use the method below getUsername(0) to also get it.

Reply:
no it's okay, I didn't realise it was a set